airbyte-integrations/connectors/source-trello/acceptance-test-config.yml

```diff
@@ -5,40 +5,39 @@ test_strictness_level: "high"
 acceptance_tests:
   spec:
     tests:
-      - spec_path: "source_trello/spec.json"
-        backward_compatibility_tests_config:
-          disable_for_version: 0.2.0
+    - spec_path: "source_trello/spec.json"
+      backward_compatibility_tests_config:
+        disable_for_version: 0.2.0
   connection:
     tests:
-      - config_path: "secrets/config.json"
-        status: "succeed"
-      - config_path: "integration_tests/invalid_config.json"
-        status: "failed"
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
   discovery:
     tests:
-      - config_path: "secrets/config.json"
-        backward_compatibility_tests_config:
-          disable_for_version: 0.1.6
```

Comment:
I removed this because it yelled about duplicate keys - I think we can only have one? Whenever I see people updating this to disable for a new version, they remove the old disable_for.